05_stringer/shahriys/main.go

```diff
@@ -0,0 +1,20 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+var outstringer io.Writer = os.Stdout
```

Comment:
if the variable name is meant to mean "out stringer", I believe CamelCase is preferred in go code. i.e. outStringer.

Alternatively just plain old 'out' might be fine too